services/drm_service.go

```diff
@@ -0,0 +1,148 @@
+package services
+
+import (
+	"github.com/bitmovin/bitmovin-go/models"
+	"github.com/bitmovin/bitmovin-go/bitmovin"
+	"strings"
+	"encoding/json"
+	"errors"
+	"fmt"
+)
```

Comment:
I like keeping the internal library imports separated out from the open source ones, I think go fmt will alphabetize them as well.